account_invoice_blocking_payment/views/account_payment_line.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="account_payment_line_form_view">
+        <field name="name">account.payment.line.form (in silog_payment)</field>
+        <field name="model">account.payment.line</field>
+        <field name="inherit_id" ref="account_payment_order.account_payment_line_form"/>
+        <field name="arch" type="xml">
+            <field name="move_line_id" position="attributes">
+                <attribute name="domain">[('blocked', '=', False)]</attribute>
+            </field>
+        </field>
+    </record>
+
+</odoo>
```

Comment:
Do not change this domain (same reason: manually populating a payment order remains can be seen as an advanced use case where users should be able to pay a blocked invoice if they decide, for instance to do a partial payment but leave it blocked)